packages/fether-react/src/Send/TxDetails/TxDetails.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import React, { Component } from 'react';
+import BigNumber from 'bignumber.js';
+import { toWei } from '@parity/api/lib/util/wei';
+
+class TxDetails extends Component {
+  state = {
+    showDetails: false
+  };
+
+  renderCalculation = values => {
+    const { estimatedTxFee } = this.props;
+
+    const gasPriceBn = new BigNumber(values.gasPrice.toString());
+    const gasLimitBn = estimatedTxFee(values)
+      .div(gasPriceBn)
+      .div(10 ** 9)
+      .toFixed(0)
+      .toString();
+
+    return `Estimate amount of gas: ${gasLimitBn}`;
+  };
+
+  renderDetails = values => {
+    return `${this.renderCalculation(values)}\n`
+      .concat(`${this.renderFee(values)}\n`)
+      .concat(`${this.renderTotalAmount(values)}`);
+  };
+
+  renderFee = values => {
+    const { estimatedTxFee } = this.props;
+
+    return `Fee: ${estimatedTxFee(values)
+      .div(10 ** 18)
```

Comment:
Instead of div(10 ** 18), you can use the `fromWei` function from @parity/api/lib/util/wei